### ISSUE TITLE
WebDAV version seems to be treated as a boolean and integer

### ISSUE THREAD

**Issue by szaydel** [owncloud/pyocclient#229]
I think this is a bug, at least based on what I am seeing, but before offering a patch, I thought I should just raise it, perhaps this is legitimate and I just don't grok what is going on.

Here version seems to be getting set to a boolean value, and later it is treated as an integer. Is this really intentional or does this seem like a bug?
https://github.com/owncloud/pyocclient/blob/master/owncloud/owncloud.py#L334

Thanks!

**Comment by PVince81** [comment]
you're right, looks like a copy-paste issue.

if you fix it please make sure that the default you pick doesn't change the current behavior

**Comment by szaydel** [comment]
I don't mind proposing a fix. This should always be an integer, correct?

**Comment by PVince81** [comment]
yes, it's either 1 or 2

**Comment by szaydel** [comment]
Right, OK that's what I expected as well. Thanks. Once I have a few spare moments I will propose a fix.

**Comment by DeepDiver1975** [comment]
This repository is being archived and is no longer maintained. Closing this as part of archival. Thank you for your contribution.
